### PR TITLE
feat/impact-overview

### PR DESCRIPTION
**Pull request by misspelz**
image

### CODE REVIEW

File: src/sections/Impacts/internals/Overview.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <ImpactNumber amount="₦238,700" label="TOTAL DISBURSED" />
+        <ImpactNumber amount="400" label="BENEFICIARIES" />
+      </div>
+
+      <div className={styles.ImpactBtn}>
+        <NavHashLink
+          to={{
+            pathname: Texts.PATH
+          }}>
+          <Button label={'View Impact Report'} size={'medium'} type={'secondary'} />
```
Comment (edustipend-root) [inline src/sections/Impacts/internals/Overview.jsx:30]: Please add all page copy/strings like `'View Impact Report'` to the `constants.js` file

**Review by edustipend-root — CHANGES_REQUESTED** [review]
@misspelz this looks good, but just some minor comments.
Also I didn't see a `.stories`for the `ImpactNumber` component.

File: src/sections/Impacts/internals/Overview.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+const Overview = () => {
+  return (
+    <div className={styles.OverviewWrap}>
+      <Header size={'medium'} dataTest={TestId.OVERVIEW_TEXT} className={styles.overviewheader}>
+        {Texts.OVERVIEW}
+      </Header>
+
+      <div className={styles.ImpactNumbersWrap}>
+        <ImpactNumber amount="₦238,700" label="TOTAL DISBURSED" />
```
[the remark this answers is not in the document]
Comment (misspelz) [reply src/sections/Impacts/internals/Overview.jsx:17]: Okay

File: src/sections/Impacts/internals/Overview.jsx (hunk 1 above)
Comment (misspelz) [reply src/sections/Impacts/internals/Overview.jsx:30]: Okay

**Comment by misspelz** [comment]
> @misspelz this looks good, but just some minor comments. Also I didn't see a `.stories`for the `ImpactNumber` component.

Done now

**Review by edustipend-root — APPROVED** [review]

**Pull request merged**